fedimint-client-module/src/module/mod.rs

```diff
@@ -533,15 +543,33 @@ where
 
     pub fn outcome_or_updates<U, S>(
         &self,
-        operation: &OperationLogEntry,
+        operation: OperationLogEntry,
         operation_id: OperationId,
-        stream_gen: impl FnOnce() -> S,
+        stream_gen: impl FnOnce() -> S + 'static,
     ) -> UpdateStreamOrOutcome<U>
     where
         U: Clone + Serialize + DeserializeOwned + Debug + MaybeSend + MaybeSync + 'static,
         S: Stream<Item = U> + MaybeSend + 'static,
     {
-        operation.outcome_or_updates(&self.global_db(), operation_id, stream_gen)
+        use futures::StreamExt;
+        match self.client.get().operation_log().outcome_or_updates(
+            &self.global_db(),
+            operation_id,
+            operation,
+            Box::new(move || {
```

Comment:
@elsirion This is the biggest "hack" in the PR. I needed to decouple operationlog impl from interface, so I introduced a provisional `IOperationLog`. But to have it be dyn-compatible, I had to convert generic `U` to `serde_json::Value` and back. :shrug:

Reply:
Would happen sooner or later anyway, sounds a bit redundant, but not too bad.